exec/csnex/Cell.cs

```diff
@@ -300,13 +334,15 @@ public Boolean Equals(Cell x)
                 case Type.Boolean:
                 case Type.Bytes:
                 case Type.Dictionary:
-                    // Not implemenetd yet.
+                    // Not implemented yet.
                     return false;
                 case Type.Number:
                     return Number.IsEqual(Number, x.Number);
                 case Type.Object:
-                    // Not Implemented yet.
+                    // Not implemented yet.
                     return false;
+                case Type.Other:
+                    return Other.Equals(x.Other);
```

Comment:
I think you want to use `==` here, rather than `.Equals`. You don't want to do a deep value comparison, you only need to check whether the references are the same.

Reply:
Changed to `m_Other == x.m_Other`